pkg/common/cns-lib/volume/manager.go

```diff
@@ -68,9 +68,10 @@ const (
 	// maxLengthOfVolumeNameInCNS is the maximum length of CNS volume name.
 	maxLengthOfVolumeNameInCNS = 80
 	// Alias for TaskInvocationStatus constants.
-	taskInvocationStatusInProgress = cnsvolumeoperationrequest.TaskInvocationStatusInProgress
-	taskInvocationStatusSuccess    = cnsvolumeoperationrequest.TaskInvocationStatusSuccess
-	taskInvocationStatusError      = cnsvolumeoperationrequest.TaskInvocationStatusError
+	taskInvocationStatusInProgress      = cnsvolumeoperationrequest.TaskInvocationStatusInProgress
+	taskInvocationStatusSuccess         = cnsvolumeoperationrequest.TaskInvocationStatusSuccess
+	taskInvocationStatusError           = cnsvolumeoperationrequest.TaskInvocationStatusError
+	taskInvocationStatusPartiallyFailed = cnsvolumeoperationrequest.TaskInvocationStatusPartiallyFailed
```

Comment:
Add some comments here explaining what is paritiallyFailed.

Reply:
Added comment